src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -323,16 +323,13 @@ def dmsetup_splitname(dev: str) -> Dict[str, Any]:
 
 
 def is_ceph_device(lv: "Volume") -> bool:
-    try:
-        lv.tags['ceph.osd_id']
-    except (KeyError, AttributeError):
-        logger.warning('device is not part of ceph: %s', lv)
-        return False
+    osd_id = lv.tags.get('ceph.osd_id')
```

Comment:
If the previous code was correct, this can raise now AttributeError. From the commit message I understand that this was incorrect?

Since we want to handle the special "null" value in the next line, we can use t as default to simplify the check below:

    osd_id = lv.tags.get('ceph.osd_id', "null")

Reply:
> If the previous code was correct, this can raise now AttributeError. From the commit message I understand that this was incorrect?

Yerk... I rushed when opening this PR. This commit actually belongs to another branch that includes a refactor, making AttributeError impossible here.

> Since we want to handle the special "null" value in the next line, we can use t as default to simplify the check below:
> 
> ```
> osd_id = lv.tags.get('ceph.osd_id', "null")
> ```
yes. we can do that.